docs/release-notes/fix-noverify.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Bug Fixes**
+
+-  CLI: Stop using SSL "noverify" option for some API requests.
```

Comment:
This needs to be more informative. Can you say, even generally, which API requests? Why was this change needed?

Reply:
updated.